task07-consume-real-data/json-data.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary js with json related functions
+ */
+
+ function sortArray(array){
+    array.sort((a,b) => {
+        let a1 = a.name.toLowerCase(),
+            b1 = b.name.toLowerCase();
+    
+        if (a1 < b1) {
+            return -1;
+        }
+        if (a1 > b1) {
+            return 1;
+        }
+        return 0;
+    });
+}
+
+export async function fetchOnePieceData(){
+    const OnePieceUrl = 'https://static.akurey.com/trainings/OnePieceInformation.json';
+    
+    try {
+        let onePieceInfo = await fetch(OnePieceUrl)
+        .then( (response) => response.json())
```

Comment:
In this case if you use async/await you don't need to use the .then() and .catch() promise method. So you have a return in line 35 and also catch error in line 36, what is your purpose with those methods?

Reply:
Just trying to catch the error, but understood :) so i just leave the one of line 36 right?